app/frontend/styles/components/flashes.scss

```diff
@@ -68,6 +64,18 @@
   }
 }
 
+.govuk-notification--notice.govuk-notification__background {
```

Comment:
Having just searched, I am fairly sure we have no elements with the class `govuk-notification__background`, so we can get rid of these styles.

Reply:
We do! The new notification box I've added for multi-school jobs, for example, has the following classes applied: alert govuk-notification govuk-notification--notice govuk-notification__background

Reply:
Ah ok!